### PR TITLE
Update istio/api

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: pilot/pkg/proxy/envoy/v1/testdata/rds-cors-policy.json.golden [hunk 1]
```diff
@@ -80,7 +80,7 @@
        "allow_methods": "POST,GET,OPTIONS",
        "allow_headers": "content-type",
        "expose_headers": "x-custom-header",
-       "max_age": "seconds:300 ",
+       "max_age": "5m0s",
```
Comment (kyessenov) [inline pilot/pkg/proxy/envoy/v1/testdata/rds-cors-policy.json.golden]: This is a pretty serious bug by the way. @ijsnellf @costinm

Comment (ijsnellf) [inline pilot/pkg/proxy/envoy/v1/testdata/rds-cors-policy.json.golden]: Ugh, I think this should be integer seconds. In the Envoy code it maps to the header `Access-Control-Max-Age`: https://github.com/envoyproxy/envoy/blob/master/source/common/http/filter/cors_filter.cc#L76

`Access-Control-Max-Age` is supposed to be integer seconds: https://www.w3.org/TR/cors/#access-control-max-age-response-header

This needs to be fixed in v1alpha1 as well, but it shouldn't block this PR. I can create a PR to fix it.

Comment (ijsnellf) [reply pilot/pkg/proxy/envoy/v1/testdata/rds-cors-policy.json.golden]: Fix: https://github.com/istio/istio/pull/4324

Comment (kyessenov) [reply pilot/pkg/proxy/envoy/v1/testdata/rds-cors-policy.json.golden]: Yeap. It's an old bug.

**Comment by kyessenov** [comment]
Merging since build/test passed after vendor SHA change.

**Pull request merged**